common/bin/oo-diagnostics

```diff
@@ -1204,6 +1204,33 @@ class OODiag
 
   end
 
+  def test_auth_keys_salt_failure
+    log_dir = "/var/log/openshift/broker/"
+
+    if Rails.env.development?
+      mode = "development"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
+    else
+      mode = "production"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
+    end
+
+    unless msg_log.empty?
+      do_warn <<-AUTH_ERROR
+      Detected an '401 Unauthorized' error on the #{mode} log. This error
+      may indicate inconsistent authentication keys or salts among brokers.
+      Please follow below steps to fix the issue:
+      1. Manually synchronize the authentication keys and salt
```

Comment:
This is rather minor, but could you include " in the broker's configuration" to indicate to the user where the keys need to be synchronized?